actions/post_utils.js

```diff
@@ -53,14 +53,10 @@ export function lastPostActions(post, websocketMessageProps) {
 
         // Need manual dispatch to remove pending post
         dispatch({
-            type: PostTypes.RECEIVED_POSTS,
+            type: PostTypes.RECEIVED_NEW_POST,
```

Comment:
The reason why we see few posts in channel is because we add posts ids to `postsInChannel` array when a new message arrives.

The redux changes here keeps a check to see if the `postsInChannel` exists and adds the postId based on that.

Here the event `lastPostActions` is called by `completePostReceive` which is called on newPosts event. We are emitting that post as `PostTypes.RECEIVED_POSTS` instead here i am changing this to be `NEW_POST` so we can check in mm-redux.

`PostTypes.RECEIVED_POSTS` does not have the check on mm-redux as it would be really hard to see if we should skip storing the postIds. so whenever we emit `RECEIVED_POSTS` we would be adding the posts ids to channel